### ISSUE TITLE
getOoxml() returns incorrect markup for certain toggle properties in newly-converted DOCX

### ISSUE THREAD

**Issue by sidetracklegal** [OfficeDev/office-js#1971]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

getOoxml() appears to return incorrect markup for certain toggle properties (<w:i>, <w:b>, etc.) immediately after converting an open Word .DOC to .DOCX.

## Your Environment

* Platform: PC desktop
* Host: Word (desktop)
* Office version number: 2106 (Build 14131.20278)
* Operating System: Windows 10 Pro 20H2 (19042.1052)
* Browser (if using Office on the web): N/A
 

## Steps to reproduce

1. Open [sample .DOC file](https://1drv.ms/u/s!ApaRqSFRQlEJdsGujqnAZf92zcM?e=KqTlzl) in Word (desktop)
2. File -> Info -> Convert. Hit Save, but do NOT close the document.
3. Call getOoxml() on the lone text range [EDIT: being sure to EXCLUDE the hidden paragraph mark from the target range].

The OOXML returned under these circumstances will incorrectly have the **<w:i/>** tag applied to the range, making the text italicized, despite the text being UNitaliziced in the document itself. Upon inspection of the actual OOXML of the saved .DOCX file, the tag instead appears as **<w:i val="0">**, which is of course correct/consistent with the original content of the .DOC. So the "_val="0"_" appears to be getting dropped/omitted by getOoxml() under these circumstances, leading to incorrect results. I have observed this affecting other similar tags (<w:b>, <w:u>, etc.) as well.

Curiously, if you CLOSE the document after Step 2, reopen it and THEN call getOoxml(), the returned markup in this scenario includes no <w:i> element at all (which is correct/consistent with the original file contents). This is despite the underlying OOXML of the file not having changed in the interim. So the issue only appears to arise when calling getOoxml() **immediately** after converting from .DOC to .DOCX (i.e., without closing/reopening the document first).

**Comment by AbidRahman-MSFT** [comment]
Hey @sidetracklegal - thanks for reporting the issue. We have created an internal bug (5483256) to track the progress of the investigation and will keep you up to date on any progress.

**Comment by yihuax-msft** [comment]
I can no longer repro this issue. @sidetracklegal , do you still see it? 

In the OOXML I get, I either see <w:i val="0"/> or <w:i val="off"/>. 

By the way, it is not the API issue. I must have been the Word's native error, which looks being fixed. Thanks.

**Comment by sidetracklegal** [comment]
Apologies for the delay- I am still able to reproduce, but have discovered an additional necessary detail in the repro steps: in particular, when defining the range to call getOoxml() on (in this case, presumably on the sole paragraph in the sample document), the range **must NOT include the hidden paragraph mark** in order for the issue to arise. So in other words, the issue arises if defining the target range as something like p.getRange("Start").expandToOrNullObject(p.getRange("**End**")) [where p is the paragraph in question]: if the end of the range is specified as "**After**" instead of "End", the issue does not arise.

**Comment by yihuax-msft** [comment]
Yes, I can still see it. Let me transfer the problem to another engineering team who has the knowledge on this issue. Thanks.

**Comment by yihuax-msft** [comment]
Internal bug is 5483265.
